Tensile/Source/lib/include/Tensile/ContractionProblemProperties.hpp

```diff
@@ -27,6 +27,7 @@
 #pragma once
 
 #include <Tensile/ContractionProblem.hpp>
+#include <Tensile/ContractionSolution.hpp>
```

Comment:
am I just missing why this include needed to be added?

Reply:
good catch, that was a relic of when the feature properties were in this file

Reply:
R.e. overlap, I removed the tile calc functions (`numTiles<x>`) completely, they were too specific to MLFeature, and actually it seemed clearer to just have no function. `tileGranularity` was changed to `computeGranularity` and used in a few more places.